includes/customizer/class-wc-shop-customizer.php

```diff
@@ -251,6 +265,30 @@ public function add_scripts() {
 		<?php
 	}
 
+	/**
+	 * For FSE themes add a "Customize WooCommerce" link to the Appearance menu.
+	 *
+	 * FSE themes hide the "Customize" link in the Appearance menu. In WooCommerce we have several options that can currently
+	 * only be edited via the Customizer. For now, we are thus adding a new link for WooCommerce specific Customizer options.
+	 * For further details see pca54o-1zX-p2.
```

Comment:
Sorry, but we probably could remove this line, it's an open source project, for me doesn't make sense to link into a private post that most people can't access.

Reply:
Sure 👍